src/dialog/README.md

```diff
@@ -1 +1,55 @@
-# Dialog widget
+# @dojo/widgets/dialog/Dialog widget
+Dojo 2's `Dialog` component can be used to show content inside of a window over top of application content. It provides default styling for a titlebar, content area, underlay, and a close button, and will respond to different screen sizes responsively.
```

Comment:
grammar nit: inside of a window over top of  -> inside ~of~ a window, over *the* top of